.github/workflows/update_frontend.yml

```diff
@@ -0,0 +1,73 @@
+name: Update frontend
+
+on:
+  schedule: # once a day
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  check-version:
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ steps.check_version.outputs.skip || steps.check_existing_pr.outputs.skip }}
+      latest_tag: ${{ steps.latest_frontend_version.outputs.latest_tag }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+      - name: Get latest frontend release
+        id: latest_frontend_version
+        uses: abatilo/release-info-action@v1.3.3
+        with:
+          owner: home-assistant
+          repo: frontend
+      - name: Check if version is up to date
+        id: check_version
+        run: |
+          SUPERVISOR_VERSION=$(cat .ha-frontend-version)
+          LATEST_VERSION=${{ steps.latest_frontend_version.outputs.latest_tag }}
+          echo "SUPERVISOR_VERSION=$SUPERVISOR_VERSION" >> $GITHUB_ENV
+          echo "LATEST_VERSION=$LATEST_VERSION" >> $GITHUB_ENV
+          if [[ ! "$SUPERVISOR_VERSION" < "$LATEST_VERSION" ]]; then
+            echo "Frontend version is up to date"
+            echo "skip=true" >> $GITHUB_OUTPUT
+          fi
+      - name: Check if there is no open PR with this version
+        if: steps.check_version.outputs.skip != 'true'
+        id: check_existing_pr
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          PR=$(gh pr list --state open --base main --json title --search "Autoupdate frontend to version $LATEST_VERSION")
+          if [[ "$PR" != "[]" ]]; then
+            echo "Skipping - There is already a PR open for version $LATEST_VERSION"
+            echo "skip=true" >> $GITHUB_OUTPUT
+          fi
```

Comment:
Could we treat this more like dependabot does with a dependency bump? Where there's never more then 1 PR open for "Frontend update". If another version of frontend is released and there is already a "frontend update" PR open then either that one is re-made for the new version or it is closed and a new PR opened. Might be easier to use tags to identify a "frontend update" PR then title search with the specific version.

@agners would that work for you? I'm fine with every frontend release becoming a PR if there's always just one. I think you're right that it might get noisy though if each frontend release results in a distinct PR.

Reply:
There will be only one PR at a time, this check is just if the PR is already there since we check it everyday. And in a non beta time there will be just once a week a new frontend release.

The PR is updating if there is a new version. Checkout the landingpage PR where the same workflow is already running: 

- home-assistant/landingpage#135